src/toc.ts

```diff
@@ -163,7 +163,7 @@ function removeSectionNumbers() {
         return;
     }
     const doc = editor.document;
-    const toc = getAllRootHeading(editor.document);
+    const toc: readonly Readonly<IHeadingBase>[] = getAllRootHeading(doc, false, false);
```

Comment:
Do we need default values for the last two parameters? It is not that intuitive from the name `getAllRootHeading`.

I can see that they were optional and you changed your mind later.

Reply:
Let me take a closer look tonight.